modules/ROOT/pages/configure-monitoring-cloudhub.adoc

```diff
@@ -49,8 +49,8 @@ Anypoint Monitoring non-AM images receive patch updates as soon as they are avai
 | Metrics and logs
 
 | Anypoint Platform (CloudHub)
-| 3.8.7-AM
-| Logs
+| 3.8.7
+| Metrics and Logs
```

Comment:
Should we remove any info that has to do with -AM runtimes in the paragraph above?
> Anypoint Monitoring non-AM images receive patch updates as soon as they are available, while -AM images receive the latest patch updates only once per month. It is recommended that you switch to a non-AM image if your production environment requires a Mule runtime engine patch.

Reply:
Or simply mention that customers who still use -AM runtimes, should switch non-AM runtimes ASAP?

Reply:
Are you deprecating -AM runtimes? If so, let's mention that they need to switch.

Reply:
yes, we have deprecated -AM runtimes. They should no longer be available for new deployment.

Reply:
Alright, I will make a note of that.